lib/signaling/v2/room.js

```diff
@@ -225,11 +239,24 @@ class RoomV2 extends RoomSignaling {
     };
   }
 
+  /**
+   * @private
+   */
+  _maybeAddBandwidthProfile(update) {
+    const { bandwidthProfile, bandwidthProfileRevision } = this.localParticipant;
+    if (bandwidthProfile && this._lastBandwidthProfileRevision < bandwidthProfileRevision) {
```

Comment:
curious - why do we need to maintain the `_lastBandwidthProfileRevision` - What if we published the bandwithProfile with every update ? It would increase a payload a bit, but would make our code simpler.

Reply:
@makarandp0 That was how I had implemented it at first. But it doesn't make sense to keep sending the payload even if it hasn't changed.